examples/ola-streaming-client.cpp

```diff
@@ -41,6 +42,11 @@ DEFINE_s_string(dmx, d, "", "Comma separated DMX values to send, e.g. "
 DEFINE_s_uint32(universe, u, 1, "The universe to send data for");
 DEFINE_uint8(priority, ola::dmx::SOURCE_PRIORITY_DEFAULT,
              "The source priority to send data at");
+DEFINE_s_default_bool(universe_from_stdin, s, false,
+                      "Also read the destination universe number from STDIN "
+                      "when reading DMX data from STDIN. The universe number "
+                      "must precede the channel values, and be delimited "
+                      "by whitespace. E.g. 1 0,255,128 2 0,255,127.");
```

Comment:
Grammar and clarity.
```suggestion
                      "by whitespace, e.g. 1 0,255,128 2 0,255,127");
```

Reply:
Will this break if I'm just trying to set the first channel of each universe?
`1 255 2 0 5 255`